### PR TITLE
[VEL-2319] Convert to pnpm

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

Migrate package manager from yarn to pnpm.

### 🧑‍💻 Developer Heads Up

⚡ Since we are using [Ember Octane](https://blog.emberjs.com/octane-is-here/) now:
* Feel free to migrate existing components to Glimmer Components.
* Write new ones exclusively in it.

Useful Resource : [Ember Octane vs Classic Cheat Sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/)

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [ ] Added/updated documentation
- [x] Properly labeled

### CODE REVIEW

File: addon/helpers/log-activity.ts [hunk 1]
```diff
@@ -2,20 +2,23 @@ import Helper from '@ember/component/helper';
...
+    run(() => {
+      assert(
+        '[helper][log-activity] An action is required for this helper to work properly.',
+        typeof action === 'function'
+      );
+      assert(
+        '[helper][log-activity] An actionDescription needs to be passed for the activity-log to make sense.',
+        typeof actionDescription === 'string'
+      );
+    });
```
Comment (phndiaye) [inline addon/helpers/log-activity.ts:21]: Fixes flacky tests

File: package.json [hunk 2]
```diff
@@ -75,31 +72,25 @@
     "@types/sinon": "^10.0.6",
     "@types/tether": "^1.4.6",
     "@typescript-eslint/parser": "^5.9.1",
-    "@upfluence/oss-components": "^3.x",
-    "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "^3.0.0",
+    "@upfluence/oss-components": "github:upfluence/oss-components#pn/chore/convert-to-pnpm",
```
Comment (JulienVannier66) [inline package.json]: You can use current version

**Review by JulienVannier66 — APPROVED** [review]
Otherwise LGTM

File: addon/helpers/log-activity.ts (hunk 1 above)
Comment (Miexil) [reply addon/helpers/log-activity.ts:21]: LIAR ! 😂 
I don't believe in any fix for the param assertions ^^

**Review by Miexil — APPROVED** [review]

**Pull request merged**